deps/build.jl

```diff
@@ -18,7 +18,7 @@ sundialslibs = enable_sensitivities ? [cvodes, idas, kinsol, nvecserial] : [cvod
 sundialsver = "sundials-2.5.0"
 provides(Sources, URI("http://ftp.mcs.anl.gov/pub/petsc/externalpackages/$sundialsver.tar.gz"), sundialslibs)
 
-provides(Binaries, URI("http://sourceforge.net/projects/juliadeps-win/files/$sundialsver.7z"),
+provides(Binaries, URI("https://cache.e.ip.saba.us/https://bintray.com/artifact/download/tkelman/generic/$sundialsver.7z"),
```

Comment:
It might not be, I just haven't decided yet which is going to be more reliable overall, the cache site or bintray by itself.